website/docs/cloud-docs/workspaces/settings/run-triggers.mdx

```diff
@@ -30,13 +30,19 @@ Creating run triggers requires admin access to the workspace. You must also have
 
 Under the "Source Workspaces" section, select the workspace you would like to connect as your source and click "Add workspace". You now have a run trigger established with your source workspace. Any run from that source workspace which applies successfully will now cause a new run to be queued in your workspace.
 
--> **Note:** Runs initiated in your workspace as the result of a run trigger connection to a source workspace will not auto-apply, regardless of your [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply) setting selection. You will need to manually apply these runs.
+<!-- BEGIN: TFC:only name:auto-apply-run-trigger -->
+
+## Run Triggers Auto-Apply Setting
+
+Runs initiated by run trigger will not auto-apply unless the "Auto-apply run triggers" setting is enabled. This setting operates independently of the primary workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) setting.
```

Comment:
I'm assuming the UI actually says "Auto-apply run triggers"? If so then we can bold it like so:
```suggestion
Runs initiated by a run trigger do not auto-apply unless you enable the **Auto-apply run triggers** setting. This setting operates independently of the primary workspace [auto-apply](/terraform/cloud-docs/workspaces/settings#auto-apply) setting.
```